stablehlo/dialect/StablehloOps.cpp

```diff
@@ -83,17 +83,17 @@ namespace stablehlo {
 namespace {
 void createArgs(ArrayRef<OpAsmParser::UnresolvedOperand> operands,
                 ArrayRef<Type> types,
-                SmallVector<OpAsmParser::Argument>& args) {
+                SmallVector<OpAsmParser::Argument> &args) {
```

Comment:
Is it intended that we switched from Google style to LLVM style for ampersands and stars? Let's perhaps keep it out of scope for this CL?

Reply:
I think this happened because the code in its current state is not google formatted. I reverted this one change and ran the following:

```
$ clang-format -i -style=google stablehlo/dialect/StablehloOps.cpp -dry-run -Werror
stablehlo/dialect/StablehloOps.cpp:86:51: error: code should be clang-formatted [-Wclang-format-violations]
                SmallVector<OpAsmParser::Argument>& args) {
                                                  ^
stablehlo/dialect/StablehloOps.cpp:86:52: error: code should be clang-formatted [-Wclang-format-violations]
                SmallVector<OpAsmParser::Argument>& args) {
                                                   ^
```

Reply:
Fixed. As discussed offline, google styling uses the most common pattern for `&/*` placement. Reverting these changes and running the format command fixed this.